driver/src/main.rs

```diff
@@ -97,11 +42,12 @@ fn main() {
 		let received = rx.recv().unwrap();
 		println!(": {}", received);
 
-		// initializing all needed web3 variables
 		let db_host = env::var("DB_HOST").unwrap();
-		let db_port = env::var("DB_PORT").unwrap();
-		let client = Client::connect(&db_host, db_port.parse::<u16>().unwrap())
-			.expect("Failed to initialize standalone client");
+        let db_port = env::var("DB_PORT").unwrap();
+		let db_instance = db_interface::db_interface::DbInterface::new(db_host, db_port).unwrap_or_else(|err| {
+      		println!("Problem creating DbInterface: {}", err);
+			process::exit(1);
```

Comment:
Do you want to return here instead of exiting? If you really want to exit, I think we should `panic!` instead.